### PR TITLE
completed issue #1065 with code and tests

### PR DESCRIPTION
**Pull request by rubygeek**

### CODE REVIEW

File: src/SeleniumLibrary/keywords/element.py [hunk 1]
```diff
@@ -335,6 +335,36 @@ def element_text_should_be(self, locator, expected, message=None, ignore_case=Fa
                            % (locator, expected, before_text))
             raise AssertionError(message)
 
+    @keyword
+    def element_text_should_not_be(self, locator, not_expected, message=None, ignore_case=False):
+        """Verifies that element ``locator`` contains exact text ``expected``.
```
Comment (aaltat) [inline src/SeleniumLibrary/keywords/element.py]: The doc string contains few copy paste errors. Please also see also line 351 and 353.

**Review by aaltat — CHANGES_REQUESTED** [review]
There a few doc string fixes. Other than that, code looks good.

**Comment by aaltat** [comment]
I did restart Travis. Code looks good from mobile screen but I need to take also a look from the PC screen.

**Comment by rubygeek** [comment]
Updated per comments

**Review by aaltat — APPROVED** [review]

**Comment by aaltat** [comment]
I made minor enhancements to the documentation. Once the Travis proves that I did not break anything with out PR, this is ready to merged.

**Pull request merged**